test/integration/stats_integration_test.cc

```diff
@@ -335,6 +361,20 @@ TEST_P(ClusterMemoryTestRunner, MemoryLargeHostSizeWithStats) {
   // 'release' builds, where we control the platform and tool-chain. So you
   // will need to find the correct value only after failing CI and looking
   // at the logs.
+  //
+  //
```

Comment:
nit: extra new line. Also, can we de-dup the same comment in the 3 places and just refer back to it in a single place? Up to you if you want to put it in stats.md, higher in the file, etc.